docs/pm/methods/jtbd-questioning.md

```
@@ -0,0 +1,260 @@
+- What's the threshold for change?
+- What would make the switch worth the effort?
+
+---
+
+### 4. Success Criteria Questions
+
+**Purpose:** Understand how users measure if the job is done well
+
+**Questions to ask:**
+
+**"How do users know the job is done well?"**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)